consensus/istanbul/backend/announce.go

```diff
@@ -547,6 +606,13 @@ func (sb *Backend) generateEncryptedEnodeURLs(enodeURL string, queryEnodeDestAdd
 func (sb *Backend) handleQueryEnodeMsg(addr common.Address, peer consensus.Peer, payload []byte) error {
 	logger := sb.logger.New("func", "handleQueryEnodeMsg")
 
+	// Since this is a gossiped messaged, mark that the peer gossiped it and check to see if this node already gossiped it
```

Comment:
This looks like a merge error or something similiar.  It duplicates previously existing identical code right below it (lines 618-623)

Reply:
Nice catch!